.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,13 @@
 ## Title
 
+<!-- Update the following example to be relevant -->
+
 PHP version: `8.1`
-Sniff name: `Silverstripe.Sniffs.Extension.DisallowOwnerPropertySniff`
+Proposed Sniff name: `Silverstripe.Sniffs.Extension.DisallowOwnerPropertySniff`
 
 - [ ] ReadMe updated with Sniff name and link to docs in Wiki
 - [ ] Unit test(s) added
-- [ ] Disallowed/Allowed examples added
+- [ ] Disallowed/Accepted examples added
```

Comment:
[nitpick] Disallowed/Allowed is the common wording.

Reply:
True.

Felt awkward reading.
Can update soon

Reply:
what do you mean awkward :) ?

Reply:
Just the language, subtitles between allowed and accepted, and reading back to back it felt better to swap but it doesn't really matter. will change it back for now.